### PR TITLE
Add fjäråskupan documentation

### PR DESCRIPTION
**Pull request by elupus**
## Proposed change

Add support for fjäråskupan kitchen fans https://fjaraskupan.se/ that are bluetooth equipped. Allow control of lights, fan and monitoring of their carbon and grease filters.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/53140 https://github.com/home-assistant/core/pull/54920 https://github.com/home-assistant/core/pull/54918 https://github.com/home-assistant/core/pull/54921
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2697

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fjaraskupan.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Fjäråskupan
+description: Instructions on how to configure fjäråskupan integration.
+ha_category:
+  - Binary Sensor
+  - Fan
+  - Light
+ha_release: 2021.7.2
```
Comment (klaasnicolaas) [inline source/_integrations/fjaraskupan.markdown]:
```suggestion
ha_release: 2021.8
```
Or later, but we never introduce new integrations in patch releases 😉

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/fjaraskupan.markdown (hunk 1 above)
Comment (elupus) [reply source/_integrations/fjaraskupan.markdown]: tTrue that. not too likely it will hit .8 anyway :). But let's keep it at that for now.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/fjaraskupan.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - Light
+ha_release: 2021.8
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@elupus'
+ha_domain: fjaraskupan
+ha_platforms:
+  - binary_sensor
+  - fam
```
Comment (MartinHjelmare) [inline source/_integrations/fjaraskupan.markdown]:
```suggestion
  - fan
```

File: source/_integrations/fjaraskupan.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - Fan
+  - Light
+ha_release: 2021.8
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@elupus'
+ha_domain: fjaraskupan
+ha_platforms:
+  - binary_sensor
```
Comment (MartinHjelmare) [inline source/_integrations/fjaraskupan.markdown:15]: The other platforms aren't done yet.

File: source/_integrations/fjaraskupan.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+title: Fjäråskupan
+description: Instructions on how to configure fjäråskupan integration.
+ha_category:
+  - Binary Sensor
+  - Fan
+  - Light
+ha_release: 2021.8
```
Comment (MartinHjelmare) [inline source/_integrations/fjaraskupan.markdown]:
```suggestion
ha_release: 2021.9
```

File: source/_integrations/fjaraskupan.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/fjaraskupan.markdown:15]: Now they are 😄

**Review by frenck — COMMENTED** [review]
Thanks, @elupus 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**